api/openapi/core-metadata.yaml

```diff
@@ -1078,7 +1078,9 @@ paths:
     parameters:
       - $ref: '#/components/parameters/correlatedRequestHeader'
     post:
-      summary: "Accepts an arbitrary number of RequestEnvelope instances. Each RequestEnvelope has a 'content' property that holds a specific request type. This endpoint is operation agnostic will route the request to the appropriate handler implementation within the service."
+      servers:
+      - url:  http://localhost:48081/api
```

Comment:
Not sure if this had the desired effect.  Pull open document in editor.swagger.io -- don't see /batch represented differently from other endpoints:

![image](https://user-images.githubusercontent.com/464327/72799904-d8edee80-3c0b-11ea-9786-111217d91670.png)

Commented here; applies elsewhere.

Reply:
Yeah, this is not easy to find in the UI. What you have to do is click on the endpoint to get the example request/response shown. You should then see a "Try it Out" button to the right, aligned with the "Parameters" tab. Click that and then scroll down until you see the blue "Execute" button. On top of that you will see the alternate server listed in the dropdown.
![OpenAPI_Execute](https://user-images.githubusercontent.com/1342980/72808502-4a836800-3c1f-11ea-80de-ec1d7c13a320.PNG)